indra/newview/scripts/lua/test_group_chat.lua

```diff
@@ -0,0 +1,16 @@
+LLChat = require 'LLChat'
+inspect = require 'inspect'
+LLAgent = require 'LLAgent'
+popup = require 'popup'
+
+local OK = 'OK_okcancelbuttons'
```

Comment:
Should we provide aliases in the `popup` module? To be honest, I'm not sure why the notification response strings are so awkward.

Reply:
Yep, I was thinking about it too. It would be better to provide handy built-in check, instead of using this 'magic' `OK_okcancelbuttons`.